listener.go

```diff
@@ -112,23 +112,51 @@ func processSnapshot(snapshot *common.Snapshot) {
 		go transmitDeploymentResultsToServer(errResults)
 	}
 
-	// if no tables, this a startup event for an existing DB, start bundle downloads that didn't finish
+	// if no tables, this a startup event for an existing DB
 	if len(snapshot.Tables) == 0 {
-		go func() {
-			deployments, err := getUnreadyDeployments()
-			if err != nil {
-				log.Panicf("unable to query database for unready deployments: %v", err)
-			}
-			log.Debugf("Queuing %d deployments for bundle download", len(deployments))
-			for _, dep := range deployments {
-				queueDownloadRequest(dep)
-			}
-		}()
+		startupOnExistingDatabase()
 	}
 
 	log.Debug("Snapshot processed")
 }
 
+func startupOnExistingDatabase() {
+	// ensure all deployment statuses have been sent to tracker
```

Comment:
Okay, now on apid startup, it checks to see if it has pending deployments, and processes it.